.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -29,16 +29,17 @@ To be completed before the issue can be added to backlog.
 - [ ] Acceptance criteria are described
 - [ ] Any preconditions are described
 - [ ] Possible dependence on external experts is mentioned
-- [ ] The story must be understandable for the development team
+- [ ] The story must be approved and understood by maintainers
 - [ ] Any requirements for accessibility are described
 
 ### Definition of Done
 <!-- Do not change. -->
 To be completed before the issue can be marked as Done.
+- [ ] Code exists on master branch
 - [ ] Meets acceptance criteria
 - [ ] Complies with [the standard of public code](https://standard.publiccode.net/)
 - [ ] Story has been sufficiently tested
-    - [ ] 90% code coverage
+    - [ ] Maximum realistic code coverage
```

Comment:
I agree with the fact that 90% is a bit of an arbitrary number and code coverage usually says deceptively little about how many problems there still could exist, but it does convey that the code has at least been well-tested. 

On the other end, `Maximum realistic` is a bit open to interpretation (which is fair, a developer working on a feature has the best idea of when a component has been tested thoroughly) and with a percentage in the (high) nineties you do at least keep a bit of a margin for edge-cases where 100% coverage is really hard to achieve.

I feel like there is something to say for both, may I ask why you changed this?

Reply:
That's a good point. I shouldn't have included in this PR. I just ran into it and changed it because it's simply not true at the moment, and we're re-doing the DoD this sprint. I'll remove the change from this PR and write it down to discuss when rediscussing the DoD.